cpp/core/tests/ShuffleSplitTest.cc

```diff
@@ -249,14 +250,21 @@ const std::vector<std::string> SplitterTest::input_data_2 = {
 const std::vector<std::string> SplitterTest::hash_key_1 = {"[1, 2, 2, 2, 2, 1, 1, 1, 2, 1]"};
 const std::vector<std::string> SplitterTest::hash_key_2 = {"[2, 2]"};
 
+ColumnarBatch* RecordBatchToColumnarBatch(std::shared_ptr<arrow::RecordBatch> rb) {
```

Comment:
return `std::shared_ptr<ColumnarBatch>` please, otherwise crash

Reply:
This is just for test, the pointer can be used in a short time, no crash